### ISSUE TITLE
A.01.IR06.IR07.supported.crses.as.list

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-interoperable-sds#5]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The XPath contains a pipe operator which means an OR condition.
The test method should instead tell which information is relevant, mandatory and which takes precedence over the other.

/gmd:MD_Metadata/gmd:referenceSystemInfo/gmd:MD_ReferenceSystem/gmd:referenceSystemIdentifier/gmd:RS_Identifier/gmd:code/(gmx:Anchor/@xlink:href|gco:CharacterString)

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the future work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
